octomap_generate:
  ros__parameters:
    resolution: 0.05
    max_range: 5.0
    occupancy_threshold: 0.7
    hit_probability: 0.7
    miss_probability: 0.4
    sensor_frame: "camera_color_frame"